README.md

```diff
@@ -66,6 +67,33 @@ templateFinder.templatePatterns = [
 ]
 ```
 
+## `git` Package
+
+This package provides some git and version information to the `renderDocsPocessor` that is available
+in the templates. This code as it is was made for the angular.js document generation, including some
+custom logic for special versions. However, any of the services can be overridden with custom
+behavior.
+
```

Comment:
We should explain that the git information is made available to the template via `extraData.git`